.github/workflows/codegen.yaml

```
@@ -40,17 +40,18 @@ jobs:
         run: |
           make bundle
       - name: Ensure there is no diff in bundle
+        # --ignore-matching-lines='.*createdAt:.*' added to ignore bundle differences of timestamp that is generated by upgraded operator-sdk
         run: |
-          git diff --exit-code -- .
+          git diff --ignore-matching-lines='.*createdAt:.*'
```

```suggestion
          git diff --ignore-matching-lines='.*createdAt:.*' --exit-code -- .
```
Lets add the new field instead of replacing it. I feel exit-code is required for exiting with proper error code on comparison failure.